app/helpers/mustache_helper.rb

```diff
@@ -46,15 +46,21 @@ def version
     { is_alpha: true }
   end
 
+  def js_version
+    Rails.application.config.x.js_version || ''
+  end
+
   def js_variables
-    "var js_path='" + styleguide_path('/js/dist/') + "';"
+    'var js_path="' + styleguide_path('/js/dist/') + '"; ' +
+    'var require = {"urlArgs": "' + js_version  + '"};'
```

Comment:
Unnecessary spacing detected.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.